### PR TITLE
UPDATE: Maxwell-Icepak Example

### PR DESCRIPTION
**Pull request by Devin-Crawford**
- Updated some of the text cells. Added more description.
- Added a cell to demonstrate how to query model units.
- Create variables defining geometry to avoid entering the same values twice.
- Added a cell describing how to fit the view in a Notebook.
- Add more properties to the solution setup in Maxwell.

# Pull Request Template

## Description
Please provide a brief description of the changes in this pull request.

## Checklist
Please complete the following checklist before submitting your pull request:
- [x] I have followed the example template and guide lines to add/update an example.
- [x] I have tested the example locally and verified that it is working with the latest version of AEDT.
- [x] I have verified that these changes to the best of my knowledge do not introduce any security vulnerabilities.

### CODE REVIEW

File: examples/low_frequency/multiphysics/maxwell_icepak.py [hunk 1]
```diff
@@ -12,32 +12,40 @@
...
 AEDT_VERSION = "2024.2"
 NG_MODE = False  # Open AEDT UI when it is launched.
 
 # ## Create temporary directory
 #
-# Create a temporary directory where downloaded data or
-# dumped data can be stored.
-# If you'd like to retrieve the project data for subsequent use,
-# the temporary folder name is given by ``temp_folder.name``.
+# Create a temporary working directory.
```
Comment (gmalinve) [inline examples/low_frequency/multiphysics/maxwell_icepak.py:32]: @Devin-Crawford @Samuelopez-ansys @SMoraisAnsys do we want to change description of the common sections? right now for the temporary directory we have the following:

"# ## Create temporary directory
#
# Create a temporary directory where downloaded data or
# dumped data can be stored.
# If you'd like to retrieve the project data for subsequent use,
# the temporary folder name is given by ``temp_folder.name``"

File: examples/low_frequency/multiphysics/maxwell_icepak.py [hunk 2]
```diff
@@ -53,35 +61,66 @@
 )
 # -
 
+# ### Units
```
Comment (gmalinve) [inline examples/low_frequency/multiphysics/maxwell_icepak.py:64]: @Devin-Crawford @SMoraisAnsys @Samuelopez-ansys we should also agree on the heading levels.
If I'm not wrong, in all the other examples, each section starts with a heading level 2 (##), do we want to change it to 3 (###)?

File: examples/low_frequency/multiphysics/maxwell_icepak.py (hunk 1 above)
Comment (Devin-Crawford) [reply examples/low_frequency/multiphysics/maxwell_icepak.py:32]: I think the new wording is better so we probably should update it everywhere. If you agree I'll create another branch/PR and only do that.

File: examples/low_frequency/multiphysics/maxwell_icepak.py [hunk 3]
```diff
@@ -122,80 +159,90 @@
...
+setup.props["MaximumPasses"] = 4
+setup.props["PercentError"] = 0.5
+setup.props["MinimumConvergedPasses"] = 2
+
+# ## Run the Maxwell 3D analysis
+
 m3d.analyze_setup("Setup1")
 
-# ## Postprocess
+# ## Postprocessing
```
[the remark this answers is not in the document]
Comment (Devin-Crawford) [reply examples/low_frequency/multiphysics/maxwell_icepak.py:194]: @SMoraisAnsys , @gmalinve : The header levels seem to be inconsistent. Just submitted #265

File: examples/low_frequency/multiphysics/maxwell_icepak.py (hunk 1 above)
Comment (SMoraisAnsys) [reply examples/low_frequency/multiphysics/maxwell_icepak.py:32]: Sure you can do that, and we'll discuss publicly on which content fits the most :)

Comment (Devin-Crawford) [inline examples/low_frequency/multiphysics/maxwell_icepak.py:32]: I will do a separate PR to update the templates.

**Review by Samuelopez-ansys — APPROVED** [review]

**Review by SMoraisAnsys — APPROVED** [review]

**Pull request merged**